threescale_api/auth.py

```diff
@@ -19,11 +19,10 @@ def __call__(self, r):
         credentials = self.credentials
 
         if self.location == "authorization":
+            credentials = list(credentials.values())
             auth = requests.auth.HTTPBasicAuth(*credentials)
```

Comment:
Why is it necessary to create a list instance - `values()` should return iterable or am I mistaken?